source/SkiaSharp.Views/SkiaSharp.Views.Shared/GlesInterop/Gles.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if !__WATCHOS__ && !__WASM__ && !UNO_REFERENCE_API && !__MACCATALYST__
+﻿#if !__WATCHOS__ && !__WASM__ && (!UNO_REFERENCE_API || (NET6_0_OR_GREATER && (__IOS__ || __MACOS__)))
```

Comment:
not: is this change required since it makes it harder to read.

Reply:
The `UNO_REFERENCE_API` is incorrectly defined for net6 in uno, and I don't want to force an update in SkiaSharp for this. I'll add a note to the related issue.